.github/workflows/CI.yml

```diff
@@ -55,6 +55,11 @@ jobs:
       - run: cargo run --release -- --no-manual-traits ../glib-macros
         working-directory: checker
         if: matrix.rust == 'nightly' && matrix.conf.name == 'glib'
+      - name: Check two levels glib dependent
```

Comment:
it should probably use the checkout action (see few lines above)

Reply:
Indeed. Thanks @bilelmoussaoui !